CMake error! Can't create makefile

System: Ubuntu 20.04 Desktop LTS

Output message:
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
/home/flowerfell-sans/桌面/ddnet-client/AVCODEC_INCLUDEDIR
   used as include directory in directory /home/XX/桌面/ddnet
/home/flowerfell-sans/桌面/ddnet-client/AVFORMAT_INCLUDEDIR
   used as include directory in directory /home/XX/桌面/ddnet
/home/flowerfell-sans/桌面/ddnet-client/AVUTIL_INCLUDEDIR
   used as include directory in directory /home/XX/桌面/ddnet
/home/flowerfell-sans/桌面/ddnet-client/SWRESAMPLE_INCLUDEDIR
   used as include directory in directory /home/XX/桌面/ddnet
/home/flowerfell-sans/桌面/ddnet-client/SWSCALE_INCLUDEDIR
   used as include directory in directory /home/XX/桌面/ddnet
AVCODEC_LIBRARY (ADVANCED)
    linked by target "DDNet" in directory /home/XX/桌面/ddnet
AVFORMAT_LIBRARY (ADVANCED)
    linked by target "DDNet" in directory /home/XX/桌面/ddnet
AVUTIL_LIBRARY (ADVANCED)
    linked by target "DDNet" in directory /home/XX/桌面/ddnet
SWRESAMPLE_LIBRARY (ADVANCED)
    linked by target "DDNet" in directory /home/XX/桌面/ddnet
SWSCALE_LIBRARY (ADVANCED)
    linked by target "DDNet" in directory /home/XX/桌面/ddnet-client
X264_LIBRARY
    linked by target "DDNet" in directory /home/flowerfell-sans/桌面/ddnet-client

--- a/README.md
+++ b/README.md
@@ -32,21 +32,21 @@ Dependencies on Linux / macOS
 
 You can install the required libraries on your system, `touch CMakeLists.txt` and CMake will use the system-wide libraries by default. You can install all required dependencies and CMake on Debian or Ubuntu like this:
 
-    sudo apt install build-essential cmake git google-mock libcurl4-openssl-dev libssl-dev libfreetype6-dev libglew-dev libnotify-dev libogg-dev libopus-dev libopusfile-dev libpnglite-dev libsdl2-dev libsqlite3-dev libwavpack-dev python
+    sudo apt install build-essential cmake git google-mock libcurl4-openssl-dev libssl-dev libfreetype6-dev libglew-dev libnotify-dev libogg-dev libopus-dev libopusfile-dev libpnglite-dev libsdl2-dev libsqlite3-dev libwavpack-dev python libx264-dev libavfilter-dev libavdevice-dev libavformat-dev libavcodec-extra libavutil-dev
 
 Or on CentOS, RedHat and AlmaLinux like this:
 
-    sudo yum install gcc gcc-c++ make cmake git python2 gtest-devel gmock-devel libcurl-devel openssl-devel freetype-devel glew-devel libnotify-devel libogg-devel opus-devel opusfile-devel pnglite-devel SDL2-devel sqlite-devel wavpack-devel
+    sudo yum install gcc gcc-c++ make cmake git python2 gtest-devel gmock-devel libcurl-devel openssl-devel freetype-devel glew-devel libnotify-devel libogg-devel opus-devel opusfile-devel pnglite-devel SDL2-devel sqlite-devel wavpack-devel libx264-devel ffmpeg-devel
 
 Or on Arch Linux like this:
 
-    sudo pacman -S --needed base-devel cmake curl freetype2 git glew gmock libnotify opusfile python sdl2 sqlite wavpack
+    sudo pacman -S --needed base-devel cmake curl freetype2 git glew gmock libnotify opusfile python sdl2 sqlite wavpack x264 ffmpeg
 
 There is an [AUR package for pnglite](https://aur.archlinux.org/packages/pnglite/). For instructions on installing it, see [AUR packages installation instructions on ArchWiki](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages).
 
 On macOS you can use [homebrew](https://brew.sh/) to install build dependencies like this:
 
-    brew install cmake freetype glew googletest opusfile SDL2 wavpack
+    brew install cmake freetype glew googletest opusfile SDL2 wavpack x264 ffmpeg
 
 If you don't want to use the system libraries, you can pass the `-DPREFER_BUNDLED_LIBS=ON` parameter to cmake.
 
@@ -96,8 +96,6 @@ Whether to enable client compilation. If set to OFF, DDNet will not depend on Cu
 
 * **-DVIDEORECORDER=[ON|OFF]** <br>
 Whether to add video recording support using FFmpeg to the client. Default value is ON.
-
-Dependencies needed on debian: `libx264-dev libavfilter-dev libavdevice-dev libavformat-dev libavcodec-extra libavutil-dev`
 
 * **-DDOWNLOAD_GTEST=[ON|OFF]** <br>
 Whether to download and compile GTest. Useful if GTest is not installed and, for Linux users, there is no suitable package providing it. Default value is OFF.

